data/json/recipes/recipe_armor.json

```diff
@@ -7374,27 +7374,27 @@
     "category": "CC_ARMOR",
     "subcategory": "CSC_ARMOR_STORAGE",
     "skill_used": "tailor",
-    "skills_required": [ "pistol", 1 ],
+    "skills_required": [ "gun", 3 ],
```

Comment:
Gun 2 (or even 1) should be enough. It's not like you need a lot of marksmanship knowledge to make a holster.